pkg/cmd/policy_enable.go

```diff
@@ -48,15 +52,52 @@ func newPolicyEnableCmd() *cobra.Command {
 				version = &cliArgs[1]
 			}
 
+			// Load the configuration from the user-specified JSON file into config object.
+			var config map[string]*json.RawMessage
+			if args.config != "" {
+				// config, err = loadJSONConfigFile(args.configFile)
+				config, err = loadPolicyConfigFromFile(args.config)
+				if err != nil {
+					return err
+				}
+			}
+
 			// Attempt to enable the Policy Pack.
-			return policyPack.Enable(commandContext(), args.policyGroup, backend.PolicyPackOperation{
-				VersionTag: version, Scopes: cancellationScopes})
+			return policyPack.Enable(commandContext(), args.policyGroup,
+				backend.PolicyPackOperation{
+					VersionTag: version,
+					Scopes:     cancellationScopes,
+					Config:     config,
+				})
 		}),
 	}
 
 	cmd.PersistentFlags().StringVar(
 		&args.policyGroup, "policy-group", "",
 		"The Policy Group for which the Policy Pack will be enabled; if not specified, the default Policy Group is used")
 
+	cmd.PersistentFlags().StringVar(
+		&args.config, "config", "",
+		"The file path for the policy pack configuration file")
+
 	return cmd
 }
+
+func loadPolicyConfigFromFile(file string) (map[string]*json.RawMessage, error) {
+	analyzerPolicyConfig, err := resourceanalyzer.LoadPolicyPackConfigFromFile(file)
+	if err != nil {
+		return nil, err
+	}
+
+	// Convert type map[string]plugin.AnalyzerPolicyConfig to map[string]*json.RawMessage.
+	config := map[string]*json.RawMessage{}
+	for k, v := range analyzerPolicyConfig {
+		bytes, err := json.Marshal(v)
```

Comment:
I don't think we can just marshal `v` directly.

Doing so results in JSON like:

```json
{"EnforcementLevel":"mandatory","Properties":{"foo":"bar"}}
```

The value needs to be converted into the format expected by the service. Something like:

```go
func marshalAnalyzerPolicyConfig(v AnalyzerPolicyConfig) (*json.RawMessage, error) {
	m := make(map[string]interface{})
	for k, v := range v.Properties {
		m[k] = v
	}
	if v.EnforcementLevel != "" {
		m["enforcementLevel"] = v.EnforcementLevel
	}
	b, err := json.Marshal(m)
	if err != nil {
		return nil, err
	}
	raw := json.RawMessage(b)
	return &raw, nil
}
```

Which will result in:

```
{"enforcementLevel":"mandatory","foo":"bar"}
```

I created a Go Playground to demonstrate: https://play.golang.org/p/L8GYTzTAsOR

Reply:
Correct, I noticed it structured the configurable values under the Properties key. Thanks for adding that code snippet btw. I updated to convert the values.